sql/core/src/main/scala/org/apache/spark/sql/CacheManager.scala

```diff
@@ -39,7 +39,7 @@ private case class CachedData(plan: LogicalPlan, cachedRepresentation: InMemoryR
 private[sql] class CacheManager(sqlContext: SQLContext) extends Logging {
 
   @transient
-  private val cachedData = new scala.collection.mutable.ArrayBuffer[CachedData]
+  val cachedData = new scala.collection.mutable.ArrayBuffer[CachedData]
```

Comment:
Seems no need to make this public? Didn't find this referenced anywhere other than `CacheManager`.